src/doomtype.h

```diff
@@ -99,12 +99,11 @@
 // standard and defined to include stdint.h, so include this. 
 
 #include <inttypes.h>
+#include <stdbool.h>
```

Comment:
I don't think this is a good idea. Chocolate Doom is supposed to be portable and there are still non-C99 compilers out there.

Reply:
First, we are expecting a C standard here that was reelased 26 years ago. I think we can expect people still using non-C99 compilers to invest some effort on their own to get this built.

Second, the `__bool_true_false_are_defined` symbol that we check one line later is only defined if the `<stdbool.h>` header is included, so the check doesn't even make any sense without it.